src/components/Form/CreateProfileForm/Documents.tsx

```diff
@@ -0,0 +1,187 @@
+import React, { useRef, useState } from "react";
+import { Button, Icon } from "design-react-kit";
+import { useSelector } from "react-redux";
+import FormContainer from "../FormContainer";
+import DocumentIcon from "../../../assets/icons/document.svg";
+import api from "../../../api";
+import DocumentSuccess from "../../../assets/icons/document-success.svg";
+
+type Props = {
+  handleComplete: any;
+  handleBack: any;
+};
+
+const DeleteDocument = ({ onClick }: any) => (
+  <span className="d-flex flex-row align-items-center" onClick={onClick}>
+    <Icon icon="it-delete" size="sm" color="primary" />
+    <span className="text-sm text-blue">Elimina</span>
+  </span>
+);
+
+const Documents = ({ handleComplete, handleBack }: Props) => {
+  const [isManifestationUploaded, setIsManifestationUploaded] = useState(false);
+  const [isAgreementUploaded, setIsAgreementUploaded] = useState(false);
+  const manifestationFile = useRef<any>();
+  const agreementFile = useRef<any>();
+  const agreement = useSelector((state: any) => state.agreement.value);
+
+  const handleManifestationClick = () => {
+    manifestationFile.current.click();
+  };
+
+  const handleAgreementClick = () => {
+    agreementFile.current.click();
+  };
+
+  const addFiles = (files: any, type: string) => {
+    void api.Document.uploadDocument(agreement.id, type, files[0]).then(() => {
+      switch (type) {
+        case "manifestation_of_interest":
+          setIsManifestationUploaded(true);
+          break;
+        case "agreement":
+          setIsAgreementUploaded(true);
+          break;
+      }
+    });
```

Comment:
Can we use ternary operator in this case?

Reply:
As you want but I liked more this approach for readibility.